app/controllers/v0/profile/permissions_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module V0
+  module Profile
+    class PermissionsController < ApplicationController
+      include Vet360::Writeable
+
+      before_action { authorize :vet360, :access? }
+      after_action :invalidate_cache
+
+      def create
+        write_to_vet360_and_render_transaction!(
+          'permission',
+          strip_effective_end_date(permission_params)
```

Comment:
I realize that you merely copied a pattern, but the  `strip_effective_end_date` method is unnecessary. Deleting `:effective_end_date,` is the cleaner way to handle this.  I've cleaned up the other instances of this in #3465

Reply:
Ok, thanks I see what you have done - curious why it was originally written with the strip method - but it seems it is a better practice not to include the effective end date in the params to start with so that striping is no longer necessary.

Reply:
@annaswims if you look below, yesterday I attempted to resolve a conflict with the routes.rb file - I thought the resolution process in Git would just merge that one resolved file into this feature branch - but it seems to have merged all of the master branch changes into this feature branch - AND it is giving the warning below. Can you advise how to proceed with this? I am not sure how to revert that merge if necessary. Thanks.

Reply:
it looks good to me.  Looking at the PR diff, it's still the same 25 or so files that are related to your PR.  

your PR is large, but it's a lot of swagger docs and specs so I don't think you really need to break it down.  We've been experimenting with a bot to comment on large PR's but right now consider it an optional suggestion.